x-pack/solutions/observability/plugins/observability_agent_builder/server/tools/get_change_points/get_log_change_points.ts

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AggregationsAggregationContainer } from '@elastic/elasticsearch/lib/api/types';
+import type { IScopedClusterClient } from '@kbn/core/server';
+import { orderBy } from 'lodash';
+import { z } from '@kbn/zod';
+import { ToolType } from '@kbn/onechat-common';
+import { ToolResultType } from '@kbn/onechat-common/tools/tool_result';
+import type { BuiltinToolDefinition } from '@kbn/onechat-server';
+import type { CoreSetup, Logger } from '@kbn/core/server';
+import type {
+  ObservabilityAgentPluginSetupDependencies,
+  ObservabilityAgentPluginStart,
+  ObservabilityAgentPluginStartDependencies,
+} from '../../types';
```

Comment:
Looks like a faulty merge. The types should be renamed to `ObservabilityAgentBuilderPlugin*
`